### DISCUSSION TITLE
Does  f motion works for u guys?

### DISCUSSION THREAD

**Discussion by JoseConseco** [folke/flash.nvim#15] [Q&A]
In video below I try to delete  forward to 2n   -  dfn;
https://github.com/folke/flash.nvim/assets/13521338/8ea3f7d5-52bb-4e9a-bedc-c704f1bf9144
As u can see all following n's are highlighted, and I can jump to them with ;  but delete is only removing characters up to (including)  first n.
Is this bug? Can we delete to nth occurrence of letter?  I mean te highlights are there, we can jump to them, but operator ignores them.

**Comment by folke** [comment]
Can you let me know what doesn't work specifically?

with the text: `foobar bar`

When on the first char, doing `2dfb` will delete everything up to (and including) the second `b`.

**Comment by JoseConseco** [comment]
My, bad.  I got used to HOP  behavior, where all b's would get labeled - and **operation would be performed up to n'th  occurrence** of letter. 
I forgot how build-in f,t,F,T worked - where pressing ;  - would just move cursor forward to next occurrence of letter, **without deleting (or performing other operation)** on that line.
